src/swagger.ts

```diff
@@ -82,18 +85,32 @@ function defineRoutes(paths: OpenAPIV2.PathsObject, controllers: Controllers): R
   return router;
 }
 
-export default function swagger(swaggerFile: string, controllers: Controllers): Router {
+const defaultOptions: Options = {
+  middleware: [],
+  useInternalErrorHandler: true,
```

Comment:
do we even need it here? maybe we should remove it altogether?

Reply:
I was aiming to stage it. Not sure how PG is using it, so in order to don't break everything on their end, I was trying to make it optional first, and then if they don't use it as well, we remove it.

Reply:
We've decided to remove this.